client/landing/stepper/declarative-flow/internals/steps-repository/plans/plans-wrapper.tsx

```diff
@@ -0,0 +1,234 @@
+								hideFreePlan={ hideFreePlan }
+								isInSignup={ true }
+								intervalType={ getIntervalType() }
+								onUpgradeClick={ onSelectPlan }
+								showFAQ={ false }
+								domainName={ getDomainName() }
+								customerType={ customerType }
+								plansWithScroll={ isDesktop }
+								planTypes={ planTypes }
+								flowName={ flowName }
+								showTreatmentPlansReorderTest={ false }
+								isAllPaidPlansShown={ true }
+								isInVerticalScrollingPlansExperiment={ isInVerticalScrollingPlansExperiment }
+								shouldShowPlansFeatureComparison={ isDesktop } // Show feature comparison layout in signup flow and desktop resolutions
+								isReskinned={ isReskinned }
+								isFAQCondensedExperiment={
+									experimentAssignment?.variationName === 'treatment_condensed'
+								}
+								isFAQExperiment={
+									experimentAssignment?.variationName === 'treatment_expanded' ||
+									experimentAssignment?.variationName === 'treatment_condensed'
+								}
+							/>
+						);
+					} }
+				</ProvideExperimentData>
+			</div>
+		);
+	};
+
+	const getHeaderText = () => {
+		if ( isDesktop ) {
+			return headerText;
+		}
+
+		return __( "Pick a plan that's right for you." );
+	};
+
+	const getSubHeaderText = () => {
+		const { flowName } = props;
+
+		const freePlanButton = (
+			<Button onClick={ handleFreePlanButtonClick } className="is-borderless" />
+		);
+
+		if ( flowName === NEWSLETTER_FLOW ) {
+			return hideFreePlan
+				? __( 'Unlock a powerful bundle of features for your Newsletter.' )
+				: translate(
+						`Unlock a powerful bundle of features for your Newsletter. Or {{link}}start with a free plan{{/link}}.`,
+						{ components: { link: freePlanButton } }
+				  );
+		}
+
+		return hideFreePlan
+			? __( 'Unlock a powerful bundle of features for your Link in Bio.' )
+			: translate(
+					`Unlock a powerful bundle of features for your Link in Bio. Or {{link}}start with a free plan{{/link}}.`,
+					{ components: { link: freePlanButton } }
+			  );
+	};
+
+	const plansFeaturesSelection = () => {
+		const { flowName } = props;
+
+		const headerText = getHeaderText();
+		const fallbackHeaderText = headerText;
+		const subHeaderText = getSubHeaderText();
+		const fallbackSubHeaderText = subHeaderText;
+
+		return (
+			<>
+				<StepWrapper
+					flowName={ flowName }
+					stepName={ stepName }
+					shouldHideNavButtons={ true }
+					fallbackHeaderText={ fallbackHeaderText }
+					fallbackSubHeaderText={ fallbackSubHeaderText }
+					isWideLayout={ true }
+					stepContent={ plansFeaturesList() }
+					allowBackFirstStep={ false }
+				/>
+			</>
+		);
+	};
+
+	const classes = classNames( 'plans-step', {
+		'in-vertically-scrolled-plans-experiment': isInVerticalScrollingPlansExperiment,
+		'has-no-sidebar': true,
+		'is-wide-layout': true,
+	} );
+
+	return (
+		<div className="stepper-plans">
+			<QueryPlans />
+			<MarketingMessage path="signup/plans" />
```

Comment:
This seems fishy.

Reply:
Removed `MarketingMessage`, as without a `siteId` it won't ever return any message.